metabase/metabase
# Add default update action to actions dropdown in object detail modal

Closes #32322

### Description

- [Hide whitespace](https://github.com/metabase/metabase/assets/6830683/cde5ae72-2216-4ddd-911c-68ef00bd7f70) is recommended when reviewing this PR
- Tests will be added in #32350

### How to verify

> Clicking update should open the existing basic action update form on top of the detail modal. The user can make whatever edits they wish and hit update. The update modal should close and the user should see any updated values present in the detail modal along with a confirmation toast.

### Demo

https://github.com/metabase/metabase/assets/6830683/97d990d2-e475-42e3-8397-235bf7ff0a6a
